### PR TITLE
User favorites #8

### LINKED ISSUE #8: User List Favorites
As a user,
When I visit the favorites page
it displays all my favorite locations.

### PR DESCRIPTION
**Pull request by cebarks**
#### Pivotal URL or Waffle Card number:
closes #8 

#### What does this PR do?
implements listing of user favorites

#### Where should the reviewer start?
javascript function breakdown

#### Any additional context you want to provide?
api-key is hardcoded as there is no login/register implemented yet

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,8 +1,37 @@
 import './styles.scss';
 
+const API_URL = 'https://as-sweater-weather.herokuapp.com/api/v1/'
```
Comment (mgoodhart5) [inline lib/index.js:3]: Assigning the url to a variable really cleans up all the requests nicely

File: lib/index.js [hunk 2]
```diff
@@ -1,8 +1,37 @@
...
+    .then(obj => addFavorites(obj));
+}
+
+function addFavorites(obj) {
+  for (var i = 0; i < obj.data.length; i++) {
+    getFavorite(obj.data[i].id);
+  }
+}
+
+function addFavorite(fav) {
```
Comment (mgoodhart5) [inline lib/index.js:24]: Nice implementation of this with .append.

**Review by mgoodhart5 — COMMENTED** [review]
This looks good! I'm interested to hear why you went with fetch vs ajax! Do you have a preference for any particular reason?

**Comment by cebarks** [comment]
@mgoodhart5 from what I've read fetch is just used more often? That together with being built into javascript, it seemed more helpful for the future.

**Pull request merged**